hy/completer.py

```diff
@@ -92,16 +95,14 @@ def attr_matches(self, text):
         return matches
 
     def global_matches(self, text):
-        path = PATH
-        if self.namespace:
-            path.append(self.namespace)
```

Comment:
This was adding the namespace to the global PATH variable on every completion. 
However, this code never used to be run because the namespace was `None`.